README.md

```diff
@@ -103,21 +105,46 @@ $ sudo dnf -y install nodejs
 > Don't forget to install node-cli, node-gyp packages using `$ npm install -g node-cli node-gyp` if it's not installed.<br />
 > Don't forget to do `npm init` to generate profile.json.
 
+## Platform Setup
+
 ### iOS
 
-Run Mobidex from XCode to use the simulator. First do the following:
+> Note: Can only be done on a Mac OS X Specific Setup
+
+Additional Requirements
+
+- [bundler](https://bundler.io)
+
+To run Mobidex on the iOS simulator from XCode, follow these steps.
+
+1. From the Mobidex root  directory, install all the required javascript packages.
+
+  `npm install`
```

Comment:
Maybe something like: `Make sure you've ran the setup steps above, first`.

Reply:
Will replace step 1